src/proto/primitive.proto

```diff
@@ -67,6 +68,12 @@ enum MaxAllowedSpeedMode
     STOP_COMMAND = 1;
     // Move at speed at which collisions are allowed
     COLLISIONS_ALLOWED = 2;
+    // Slow speed for retreating in ball placement
+    BALL_PLACEMENT_RETREAT = 3;
+    // Slow speed for picking the ball off the wall in ball placement
+    BALL_PLACEMENT_WALL_DRIBBLE = 4;
+    // Slower speed for dribbling
+    DRIBBLE = 5;
```

Comment:
why can't we use `DRIBBLE` for everything?

Reply:
the other two speeds are way slower than dribble, and we separate them for tunability